tests/cross_cluster_live_migration/conftest.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import pytest
+from ocp_resources.resource import get_client
+
+from utilities.constants import REMOTE_KUBECONFIG
+
+
+@pytest.fixture(scope="session")
+def remote_kubeconfig_export_path():
+    return os.environ.get(REMOTE_KUBECONFIG)
```

Comment:
if is not set `REMOTE_KUBECONFIG` you end up with the same client
```
config_file or os.environ.get("KUBECONFIG", "~/.kube/config")
```

I also think we should support contextsbecause the same kubeconfig can be used to access multiple clusters

Reply:
Added the check for the env var
Regarding the contexts, I don't think we want to use it, it may complicate things when running several jobs from the executor